src/functTest/groovy/com/bmuschko/gradle/docker/tasks/image/DockerRemoveImageFunctionalTest.groovy

```diff
@@ -74,13 +81,20 @@ class DockerRemoveImageFunctionalTest extends AbstractFunctionalTest {
                 force = true
                 targetImageId { buildImage.getImageId() }
             }
+
+			task removeImageAndCheckRemoval(type: DockerListImages) {
+				dependsOn removeImage
+    			showAll = true
+    			filters = '{"dangling":["true"]}'
+			}
+            
         """
 
         when:
-        BuildResult result = build('removeImage')
+        BuildResult result = build('removeImageAndCheckRemoval')
 
         then:
-        result.output.contains("Removing image with ID")
+        !result.output.contains("repository")
```

Comment:
@orzeh most of our tests do this so I'm not sure if we should tell folks not to. There is an open PR/idea that I put together previously, that you commented on, that I thought would be better to implement over this. I guess my thinking is that if we are going to move towards that anyway (which I think is better) lets not tell users to use this.